browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -163,8 +163,14 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 			$('#applystyle').val(state).trigger('change');
 		}
 		else if (commandName === '.uno:ModifiedStatus') {
-			if (document.getElementById('save')) {
+			const saveEle = document.getElementById('save');
+			if (saveEle) {
 				if (state === 'true') {
+					saveEle.classList.remove('saving');
+					saveEle.classList.remove('saved');
+					const saveIconEl = document.querySelector('#save img');
```

Comment:
I see similar code repeated 3 times. Can we do a followup with cleanup - sharing.
Please create new file (in TS), eg. `browser/src/control/Control.SaveIcon.ts`
and define there methods related to the save icon which will be only called from other files.

Reply:
Thanks, @eszkadev. I am working on it...